internal/querynode/flow_graph_service_time_node.go

```diff
@@ -29,8 +29,6 @@ import (
 type serviceTimeNode struct {
 	baseNode
 	loadType          loadType
-	collectionID      UniqueID
```

Comment:
The collection ID can be kept, and the collection ID can be printed out when add log

Reply:
fixed